pyro/contrib/tracking/hashing.py

```diff
@@ -0,0 +1,133 @@
+from __future__ import absolute_import, division, print_function
+
+import itertools
+from collections import defaultdict
+from numbers import Number
+
+
+class LSH(object):
+    """
+    Implements locality-sensitive hashing.
```

Comment:
How about we keep the class name `LSH` and describe a little more specifically:
> Implements locality-sensitive hashing for low-dimensional euclidean space.

Reply:
I kept the name `LSH` for now, will we have additional hashing schemes in the future?

Reply:
It's conceivable that we might track in embedding space rather than physical space, and in that case a cosine-distance random-hyperplane LSH would work better than this version. But let's address that when it arises.